.pre-commit-config.yaml

```diff
@@ -0,0 +1,32 @@
+repos:
+  - repo: https://github.com/pre-commit/pre-commit-hooks
+    rev: v4.6.0
+    hooks:
+      - id: end-of-file-fixer
+      - id: trailing-whitespace
+      - id: mixed-line-ending
+      - id: fix-byte-order-marker
+      - id: check-merge-conflict
+      - id: check-added-large-files
+
+  - repo: https://github.com/gruntwork-io/pre-commit
+    rev: v0.1.15
+    hooks:
+      - id: helmlint
+
+  - repo: https://github.com/norwoodj/helm-docs
+    rev: v1.14.2
+    hooks:
+      - id: helm-docs
+        files: (README\.md\.gotmpl|(Chart|requirements|values)\.yaml)$
+        args:
+          - --chart-search-root=.
+          - --template-files=./.github/helm-docs/_templates.gotmpl
+          - --template-files=README.md.gotmpl
```

Comment:
I'm wondering if we need this hook if we are already generating it automatically when we open a PR. I confess I prefer to generate it automatically on the workflow. Wdyt?

Reply:
Please notice that the CI pipeline uses another pre-commit config that does not use this hook. This means that you, as a contributor, can either generate the docs automatically using this pre-commit config or have the workflow generate them for you. There will be cases when you want to generate the docs locally to check and review them.

Thus, nothing is enforcing you to generate the docs locally, but you have the option to generate it locally (although you can always use `make docs` to generate the docs)

I will remove it and leave only a single pre-commit config. If you want to generate the docs locally, you can use `make docs` or run `helm-docs` manually.

Reply:
Done [7b93f2a](https://github.com/NethermindEth/helm-charts/pull/7/commits/7b93f2aefeb8329ea050eeaa9c05e45b99d08afd)